framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelHelpers.cs

```diff
@@ -0,0 +1,124 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Net.Sockets;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// Collection of constants used by the AzureQueueStorage Channel classes
+    /// </summary>
+    static class AzureQueueStorageConstants
+    {
+        internal const string EventLogSourceName = "Microsoft.Samples.AzureQueueStorage";
+        internal const string Scheme = "soap.aqs";
+        internal const string AzureQueueStorageBindingSectionName = "system.serviceModel/bindings/sampleProfileAzureQueueStorageBinding";
+        internal const string AzureQueueStorageTransportSectionName = "aqsTransport";
```

Comment:
You don't need config support at the client packages don't  support config. These two constants are related to config.

Reply:
ok. Removed them.